india_banking/india_banking/doc_events/bank_account/bank_account.py

```
@@ -77,3 +85,18 @@ def update_party_transaction_currency(doc):
+def get_bank_details_from_branch_code(branch_code):
+	url = f"https://ifsc.razorpay.com/{branch_code}"
+
+	try:
+		response = requests.get(url)
+		bank_details_json = {}
+		if response.ok:
+			bank_details_json = response.json()
+
+		return bank_details_json
+
+	except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)